src/redux/board-reducer.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { storageState } from "../util/storage";
+
+const boardInitialState = [];
+
+const boardSlice = createSlice({
+    name: "board",
+    initialState: storageState("board", boardInitialState),
+    reducers: {
+      setBoardInfo(state, action) {
+        action.payload.map(board=>{
+          state.push(board);
+        });
+      },
```

Comment:
```suggestion
      setBoardInfo(state, action) {
        action.payload.forEach(board => state.push(board));
      },
```
`map()`은 새로운 배열을 만들기 위함이고, `forEach()`는 반복문입니다.
새로운 배열이 필요한 행위가 아니니 forEach()로 바꾸면 좋을 것 같네요
[참고](https://pewww.tistory.com/12)